.circleci/config.yml

```
@@ -94,19 +94,19 @@ commands:
             echo "WASM_ENGINES = []" >> .emscripten
             test -f ~/vms/wasmtime && echo "WASMTIME = os.path.expanduser(os.path.join('~', 'vms', 'wasmtime')) ; WASM_ENGINES.append(WASMTIME)" >> .emscripten || true
             test -f ~/vms/wasmer && echo "WASMER = os.path.expanduser(os.path.join('~', 'vms', 'wasmer')) ; WASM_ENGINES.append(WASMER)" >> .emscripten || true
-            test -d ~/wasi-sdk && cp -ar ~/wasi-sdk/lib/ ~/emsdk/upstream/lib/clang/16.0.0/
+            test -d ~/wasi-sdk && cp -a ~/wasi-sdk/lib/ ~/emsdk/upstream/lib/clang/16.0.0/
             cd -
             echo "final .emscripten:"
             cat ~/emsdk/.emscripten
       - emsdk-env
       - npm-install
+  build-libs:
+    description: "Build all libraries, and freeze the cache"
```

```suggestion
    description: "Build all libraries"
```
Looks like this comment was stale/incorrect before, but might as well fix it now.